website/content/en/docs/Reference/_index.md

```diff
@@ -0,0 +1,439 @@
+---
```

Comment:
We need an automated mechanism to copy these docs. 

Something like
```
codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
        ...
	gen-crd-api-reference-docs \
		-api-dir ./pkg/apis/provisioning/v1alpha3 \
		-config $(shell go env GOMODCACHE)/github.com/ahmetb/gen-crd-api-reference-docs@v0.3.0/example-config.json \
		-out-file website/content/en/docs/Reference/_index.md \
		-template-dir $(shell go env GOMODCACHE)/github.com/ahmetb/gen-crd-api-reference-docs@v0.3.0/template
```

What happens if we're missing the top of this index file? 
```
---
title: "API Reference"
linkTitle: "API Reference"
weight: 70
---
```

Reply:
that's #532 :)

we need that "frontmatter" (in hugo terms) for the page header and nav structure to render properly.

Reply:
how about combining the output with a template and overwriting the existing page? 

```
codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
        ...
	gen-crd-api-reference-docs \
		-api-dir ./pkg/apis/provisioning/v1alpha3 \
		-config $(shell go env GOMODCACHE)/github.com/ahmetb/gen-crd-api-reference-docs@v0.3.0/example-config.json \
		-out-file website/content/en/docs/Reference/code-gen.txt \
		-template-dir $(shell go env GOMODCACHE)/github.com/ahmetb/gen-crd-api-reference-docs@v0.3.0/template
	cat <(cat website/content/en/docs/Reference/api-docs-header.txt) \
		website/content/en/docs/Reference/code-gen.txt \
		> website/content/en/docs/Reference/_index.md
```

Reply:
looks perfect. Bonus points for not creating a temp file.